mne/epochs.py

```diff
@@ -1588,6 +1588,7 @@ def _get_data(
         units=None,
         tmin=None,
         tmax=None,
+        copy=True,
```

Comment:
in this *private* method we should probably default to `copy=False` so that we're not making unnecessary copies in *internal* calls that use the private API.